package.json

```diff
@@ -20,7 +20,7 @@
     "test": "cross-env NODE_ENV=test jest",
     "test:accessibility": "./node_modules/.bin/pa11y-ci",
     "test:coverage": "cross-env NODE_ENV=test jest -i --coverage",
-    "precommit": "npm run lint && npm run test && npm run styleguide:build",
+    "precommit": "./precommit.sh && npm run lint && npm run test",
```

Comment:
why did we take the styleguide:build out of the precommit hook?

Reply:
So the build overwrites the files in the `docs` folder but doesn't add them. So you end up with unstaged files.

This way the bash script will only just yell at you for not updating the styleguide because there's no good way of re-`git add` on `precommit`.

Reply:
gotcha